app/src/play/java/com/amaze/filemanager/activities/MainActivity.java

```diff
@@ -549,11 +549,11 @@ public Void onPostExecute(Object result) {
                             if (file.isDirectory(MainActivity.this))
                                 goToMain(path);
                             else {
-                                goToMain("");
+                                goToMain(null);
```

Comment:
For anyone wanting to check if this will throw an exception, [it won't](https://github.com/TeamAmaze/AmazeFileManager/blob/5d9e9aa8e89c6a09451cc9231723522ce59da15c/app/src/play/java/com/amaze/filemanager/activities/MainActivity.java#L883).